tasks/update-cnb-dependency/commit_artifacts.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"os/exec"
+
+	"github.com/pkg/errors"
+)
+
+func CommitArtifacts(commitMessage string) error {
+	if commitMessage == "" {
+		return nil
+	}
+
+	output, err := exec.Command("git", "-C", "artifacts", "add", "buildpack.toml").CombinedOutput()
+	if err != nil {
+		return errors.Wrapf(err, "failed to add artifacts: %s", string(output))
+	}
+
+	output, err = exec.Command("git", "-C", "artifacts", "commit", "-m", commitMessage).CombinedOutput()
```

Comment:
I think this is cool for now, but we should consider using https://github.com/libgit2/git2go or something like it if things get more complicated.

Reply:
👍